### PR TITLE
Tutorial screens implementation

### PR DESCRIPTION
**Pull request by kiroj305**
Added a page view containing tutorial 1, 2, 3

### CODE REVIEW

File: lib/screens/tutorial_screen.dart [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      .map((item) => Column(
+            children: <Widget>[
+              SizedBox(height: 0.2.sh),
+              Padding(
+                  padding: EdgeInsets.symmetric(horizontal: 0.22.sw),
+                  child: Image.asset(
+                    item['image'],
+                    fit: BoxFit.contain,
+                  )),
+              if (kTutorialItems.indexOf(item) == 2) ...[
```
Comment (mustafapylons) [inline lib/screens/tutorial_screen.dart]: number "2" can be in a constant variable.

File: lib/screens/tutorial_screen.dart [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                margin: const EdgeInsets.only(bottom: 120.0),
+                padding: const EdgeInsets.symmetric(vertical: 10.0),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: indicator(),
+                ),
+              )
+              //  ),
+              ),
+          if (currentPage.round() == 2) ...[
```
Comment (mustafapylons) [inline lib/screens/tutorial_screen.dart]: number "2" can be in a constant variable.

**Review by mustafapylons — COMMENTED** [review]
few small issues, otherwise good clean code.

File: lib/screens/tutorial_screen.dart [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                        height: 0.08.sh,
+                        decoration: const BoxDecoration(
+                          image: DecorationImage(
+                              image: AssetImage(kTooltipBalloon),
+                              fit: BoxFit.contain),
+                        ),
+                        child: const Text(
+                          kWhyAppNeeded,
+                          style: TextStyle(
+                              fontSize: 12.0,
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: Use sp for text.

File: lib/screens/tutorial_screen.dart [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+                  )),
+              if (kTutorialItems.indexOf(item) == 2) ...[
+                SizedBox(
+                  height: 0.15.sh,
+                  child: Align(
+                    alignment: Alignment.bottomRight,
+                    child: GestureDetector(
+                      child: Container(
+                        alignment: Alignment.center,
+                        margin: const EdgeInsets.only(right: 10, bottom: 2),
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: use 10.w and 2.h to make it responsive.

File: lib/screens/tutorial_screen.dart [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+                        child: const Text(
+                          kWhyAppNeeded,
+                          style: TextStyle(
+                              fontSize: 12.0,
+                              fontWeight: FontWeight.w400,
+                              color: EaselAppTheme.kWhite),
+                        ),
+                      ),
+                      onTap: () {
+                        print("clicked bubble!");
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: Remove print.

File: lib/screens/tutorial_screen.dart [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+                      },
+                    ),
+                  ),
+                ),
+                RichText(
+                  text: TextSpan(
+                    style: const TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.w800,
+                        color: Color(0xFF333333)),
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: Add color in constant file.

File: lib/screens/tutorial_screen.dart [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+                    ],
+                  ),
+                )
+              ] else ...[
+                SizedBox(height: 0.15.sh),
+                Text(item['header'],
+                    style: const TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.w800,
+                        color: Color(0xFF333333)),
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: Add color in a constant file.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
Kindly solve these minor issues.

**Comment by kiroj305** [comment]
@mustafapylons , @kjawadDeveloper2 .
All issues pointed out above are now fixed.

**Review by mustafapylons — APPROVED** [review]
Looks good to me after the changes.

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks good to me.

**Pull request merged**